.github/workflows/publish_packages.yaml

```
@@ -0,0 +1,27 @@
+      - name: Extract package name from the tag
+        id: extract
+        run: |
+          TAG="${GITHUB_REF#refs/tags/}"
+          PACKAGE_NAME="${TAG%%-v*}"
+          echo "package_name=$PACKAGE_NAME" >> $GITHUB_OUTPUT
+
+      - name: Call shared publish workflow
+        uses: ./.github/workflows/publish_to_pub_workflow.yaml
+        with:
+          package_name: ${{ steps.extract.outputs.package_name }}
+          tag: ${{ github.ref }}
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)